DefInjected/ThingDef/Buildings_Security.xml

```diff
@@ -33,10 +33,10 @@
 	<TrapDeadfall.label>ловушка</TrapDeadfall.label>
 	<TrapDeadfall.description>Простая, но эффективная ловушка.</TrapDeadfall.description>
 
-	<TrapIEDBomb.label>взрывное СВУ</TrapIEDBomb.label>
-	<TrapIEDBomb.description>Самодельное взрывное устройство — артиллерийский снаряд на импровизированной растяжке.</TrapIEDBomb.description>
+	<TrapIEDBomb.label>осколочное СВУ</TrapIEDBomb.label>
+	<TrapIEDBomb.description>Самодельное взрывное устройство — артиллерийский снаряд, присоединённый к импровизированному спусковому механизму. Когда кто-нибудь активирует ловушку, снаряд взоровется. Мина сдетонирует после случайной задержки, что может позволить убежать проворному противнику.</TrapIEDBomb.description>
 
 	<TrapIEDIncendiary.label>зажигательное СВУ</TrapIEDIncendiary.label>
-	<TrapIEDIncendiary.description>Самодельное взрывное устройство зажигательного типа — артиллерийский снаряд на импровизированной растяжке.</TrapIEDIncendiary.description>
+	<TrapIEDIncendiary.description>Самодельное взрывное устройство зажигательного типа — взрывчатка, смешанная с легковоспламеняемым веществом. Имеет простой и надежный спусковой механизм в виде распылителя.</TrapIEDIncendiary.description>
```

Comment:
> The en-US language folder lacks one of these because all the original def data is in English so no English data needs to be injected. However, you can see examples from other languages like German

Здесь просто говорится что часть перевода отсутствует в английских файлах за ненадобностью и пример оформления файлов можно посмотреть в немецком, например, переводе. 
Это вовсе не значит, что немецкий перевод хороший или полный или ещё чем-то хорош, кроме того, что там есть дополнительные файлы `DefInjected`.

>Французский и даже украинский тексты тоже, по твоему, взяты с потолка?

Конечно. А есть причина так не считать?